src/eduid_common/config/tests/test_config_parser.py

```diff
@@ -3,17 +3,25 @@
 import os
 import unittest
 
-from eduid_common.config.parsers import ConfigParser
+from eduid_common.config.parsers import _choose_parser
 from eduid_common.config.parsers.etcd import EtcdConfigParser
 
 __author__ = 'lundberg'
 
+from eduid_common.config.parsers.yaml import YamlConfigParser
 
-class TestEtcdParser(unittest.TestCase):
+
+class TestInitConfig(unittest.TestCase):
     def tearDown(self):
         os.environ.clear()
 
     def test_EtcdConfigParser(self):
-        os.environ.setdefault('EDUID_CONFIG_NS', '/test/ns/')
-        parser = ConfigParser()
+        os.environ['EDUID_CONFIG_NS'] = '/test/ns/'
```

Comment:
Why do we set EDUID_CONFIG_NS to '/test/ns/' here and use ns='test' below?

Reply:
Doesn't matter - the parameter is only used as fallback if the environment variable is not set.

We could rename the argument to 'fallback_ns' or 'default_ns', but it could be conceived that a future parser would in fact not use the environment variable but rather use the ns as some kind of reference such as 'https://config.eduid.se/path' or something.